packages/dai-plugin-migrations/contracts/abiMap.json

```diff
@@ -1,3 +1,9 @@
 {
-  "TUB": "SaiTub"
+  "TUB": "SaiTub",
+  "MCD_END_1": "End",
```

Comment:
Am I correct in understanding that `_1` here means basically "from the previous MCD deployment"? Assuming that global settlement has taken place for the previous system and dai-plugin-mcd has been updated to the new one, so that its ordinary `CDP_MANAGER`, `MCD_POT`, etc. cannot be used? 

Maybe this can be made more clear by using a prefix or suffix like `MIG` or `PREV`, or at least having an explanatory comment somewhere central (perhaps in the plugin README)

Reply:
Right, not quite the 'previous' deployment; it's meant to enumerate through the different versions of each contract, whether they are from new deployments (after settlement) or just upgrades (e.g. dschief).

I'll add that to the README.